modal/sandbox.py

```diff
@@ -85,6 +85,12 @@ def _new(
                 raise InvalidError("`region` and `_experimental_scheduler_placement` cannot be used together")
             scheduler_placement = SchedulerPlacement(region=region)
 
+        if isinstance(gpu, list):
```

Comment:
The typing for Sandbox._new doesn't take in the list form, but makes sense to surface a helpful error here anyway given the misuse possibility.